pkg/project/registry/projectrequest/delegated/delegated.go

```diff
@@ -61,8 +61,8 @@ func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 	}
 
 	projectName := projectRequest.Name
-	projectDisplayName := projectName
-	projectDescription := projectDisplayName
+	projectDisplayName := ""
```

Comment:
Complete the refactor below.  May as well use the display name and description directly.